### DISCUSSION TITLE
Icon size can be set once only and never again

### DISCUSSION THREAD

**Discussion by frankfontein** [linuxmint/discussions#163] [Questions]
For a recent project I increased the size of thumbnails (icon view) in Nemo through Nemo's preferences menu. However I now find that I cannot set them back to 100% (or any other size). Moreover the size increase was only applied to the folder where I edited the preferences. Should I have expected a size increase to apply to all folders? Have I missed the obvious here in trying to get back to 100%?

**Comment by Jeremy7701** [comment]
I tried this, because I wanted big icons (temporarily) and the effect was exactly as described above.

**Comment by frankfontein** [comment]
Hi Jeremy7701 - I played for a little longer some time ago, but without luck. I conclude that it is a bug and maybe some flag (e.g. isResized == true) is not being checked or set. Nothing you and I can do about it - just wait for updates I guess.
